robot/description/urdf/constants.xacro

```diff
@@ -56,22 +54,24 @@
     <xacro:property name="diff_drive_legacymode" value="false"/>
     <xacro:property name="diff_drive_rosDebugLevel" value="na"/>
 
-    <!-- Constants for hokuyo 2D LIDAR sensor -->
+    <!-- Hokuyo 2D LIDAR sensor -->
     <xacro:property name="laser_visualize" value="false"/>
-    <xacro:property name="laser_update_rate" value="30"/>
+    <xacro:property name="laser_update_rate" value="10"/>
 
     <xacro:property name="laser_horizontal_samples" value="1080"/>
     <xacro:property name="laser_horizontal_resolution" value="1"/>
-    <xacro:property name="laser_horizontal_min_angle" value="-2.356"/>
-    <xacro:property name="laser_horizontal_max_angle" value="2.356"/>
+    <xacro:property name="laser_field_of_view" value="240"/>
+    <xacro:property name="laser_horizontal_min_angle" value="$(-laser_field_of_view*pi/180)"/>
+    <xacro:property name="laser_horizontal_max_angle" value="$(laser_field_of_view*pi/180)"/>
```

Comment:
I  believe our specific Hokuyo lidar has 270 degree fov. min_angle = -1/2 * fov  * pi/180 and max_angle = +1/2 * fov * pi/180. That is, you forgot to divide by 2

Reply:
Fixed